workbench-core/example/infrastructure/integration-tests/tests/authentication/routeHandler.test.ts

```diff
@@ -132,4 +132,33 @@ describe('authentication route handler integration tests', () => {
       ).rejects.toThrow(new HttpError(401, {}));
     });
   });
+
+  describe('logoutUser', () => {
+    it('should return the logout URL for a logged in user', async () => {
+      const adminSession = await setup.createAdminSession();
+      const origin = 'fakeOrigin';
+      const domain = setup.getSettings().get('ExampleCognitoDomainName');
+      const clientId = setup.getSettings().get('ExampleCognitoUserPoolClientId');
+
+      const { data } = await adminSession.resources.authentication.logout({ origin });
+
+      expect(data.logoutUrl).toBe(`${domain}/logout?client_id=${clientId}&logout_uri=${origin}`);
+    });
+
+    it('should return the logout URL for a logged out user', async () => {
```

Comment:
is there a test somewhere to ensure the logged out user is challenged for creds if the logged out token is presented?

Reply:
Similar to the `refreshAccessToken` tests, the token is stored in a cookie that is removed once the token expires. testing this would require a decent amount of work, and I'll add a ticket to handle this case.